src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureArea, FigureInfo {

```

Comment:
```suggestion
public class Circle extends Figure {
```
ми уже імплементували ці інтерфейси в класі Figure, тому у всіх класах наслідниках можна тепер прибрати імплементацію